packages/models-library/src/models_library/function_service/demo_units.py

```diff
@@ -27,141 +27,157 @@ def build_input(schema):
         "inputs": {
             "length": build_input(
                 {
-                    "title": "Distance",
+                    "label": "Distance",
                     "description": "Distance base unit",
                     "minimum": 0,
                     "maximum": 10,
-                    "x_unit": "m",
+                    "x_unit": "meter",
                     "type": "number",
                 }
             ),
             "time": build_input(
                 {
-                    "title": "Time",
-                    "description": "Positive time base unit",
+                    "label": "Time",
+                    "description": "Positive time in microseconds",
```

Comment:
do we need to add in the description everywhere the unit as well?
is that not implied from the x_unit field already?

Reply:
@pcrespov

Reply:
sorry I do not get it.

Reply:
discussed and cleared